src/Compressors.jl

```diff
@@ -2,6 +2,9 @@ import Blosc
 import CodecZlib
 import JSON
 
+_reinterpret(::Type{T}, x) where T = reinterpret(T, [x])
```

Comment:
This method seems to not be needed.

Reply:
it is, reinterpret doesn't work on scalars:
```julia

julia> reinterpret(UInt8, 8.)
ERROR: bitcast: argument size does not match size of target type
Stacktrace:
 [1] reinterpret(#unused#::Type{UInt8}, x::Float64)
   @ Base ./essentials.jl:438
 [2] top-level scope
   @ REPL[1]:1

julia> reinterpret(UInt8, [8.])
8-element reinterpret(UInt8, ::Vector{Float64}):
 0x00
 0x00
 0x00
 0x00
 0x00
 0x00
 0x20
 0x40
```

Reply:
But where does `_reinterpret` need to be called on a scaler? If the scalar `_reinterpret` method is deleted the tests still pass.